python/EmpireWrapper.cpp

```diff
@@ -485,7 +485,9 @@ namespace FreeOrionPython {
             .add_property("description",            make_function(&Policy::Description,         py::return_value_policy<py::copy_const_reference>()))
             .add_property("shortDescription",       make_function(&Policy::ShortDescription,    py::return_value_policy<py::copy_const_reference>()))
             .add_property("category",               make_function(&Policy::Category,            py::return_value_policy<py::copy_const_reference>()))
-            .def("adoptionCost",                    &Policy::AdoptionCost)
+            .add_property("adoptionCost",           +[](const Policy& p)                       { return p.AdoptionCost(AppEmpireID(), ScriptingContext{}); })
```

Comment:
The property does not work. The interface generator seems to ignore it an when I add a matching property manually and print `str(policy.adoptionCost)` I get `<bound method adoptionCost of <freeOrionAIInterface.policy object at 0x7ff14c069400>>` instead of a float

When I changed it to...
```suggestion
            .def("adoptionCost",                    +[](const Policy& p)                       { return p.AdoptionCost(AppEmpireID(), ScriptingContext{}); })
```
...I got a member function that works, which does the job for me.